src/puppetlabs/puppetdb/cli/shovel.clj

```diff
@@ -0,0 +1,374 @@
+(ns puppetlabs.puppetdb.cli.shovel
+  (:import [javax.jms
+            ExceptionListener
+            JMSException
+            MessageListener
+            Session
+            TextMessage
+            BytesMessage
+            ConnectionFactory
+            Connection
+            MessageConsumer
+            Queue
+            Message]
+           [java.io File ByteArrayInputStream]
+           [org.apache.activemq.broker BrokerService]
+           [org.apache.activemq.usage SystemUsage MemoryUsage]
+           [org.apache.activemq.pool PooledConnectionFactory]
+           [org.apache.activemq ActiveMQConnectionFactory ScheduledMessage])
+  (:require [clojure.tools.logging :as log]
+            [clojure.java.io :as io]
+            [me.raynes.fs :as fs]
+            [puppetlabs.puppetdb.cheshire :as json]
+            [puppetlabs.i18n.core :as i18n]
+            [puppetlabs.trapperkeeper.config :as config]
+            [puppetlabs.trapperkeeper.logging :as logutils]
+            [puppetlabs.kitchensink.core :as kitchensink]
+            [puppetlabs.puppetdb.config :as conf]
+            [puppetlabs.puppetdb.utils :as utils]
```

Comment:
I'd say just generally check this for unused stuff. I haven't checked the imports.